tools/c7n_tencentcloud/c7n_tencentcloud/query.py

```diff
@@ -279,7 +279,7 @@ def get_resource_query_params(self):
     def resources(self):
         params = self.get_resource_query_params()
         resources = self.source.resources(params)
-
+        resources = self.augment(resources)
```

Comment:
fwiw, we typically defer to the source for augment as they typically have different needs based on where the data came from.